src/ui/Base/CoveoUnderscore.ts

```diff
@@ -14,12 +14,10 @@ function setCoveoUnderscore() {
   }
 }
 
-function setTemplateSettings(instance) {
-  instance['templateSettings'] = {
-    evaluate: /(?:<%|{{)([\s\S]+?)(?:%>|}})/g,
-    interpolate: /(?:<%|{{)=([\s\S]+?)(?:%>|}})/g,
-    escape: /(?:<%|{{)-([\s\S]+?)(?:%>|}})/g
-  };
+function setTemplateSettings({ templateSettings }: { templateSettings: _.TemplateSettings }) {
+  templateSettings.evaluate = /(?:<%|{{)([\s\S]+?)(?:%>|}})/g;
+  templateSettings.interpolate = /(?:<%|{{)=([\s\S]+?)(?:%>|}})/g;
+  templateSettings.escape = /(?:<%|{{)-([\s\S]+?)(?:%>|}})/g;
 }
```

Comment:
The new version of underscore seems to have replaced every child of the `_` instance for a getter without a setter.

Reply:
Don't know if that can cause issues with clients using the underscore instance we export 🤔  Should be quite unlikely.

Is anything else unsupported?

Reply:
This, and possibly the formatting difference below, are the only differences I noticed with this version of Underscore.